src/guides/bioinformatics/filtering-and-subsampling.rst

```diff
@@ -45,17 +196,15 @@ To drop such strains, you can pass the filename to ``--exclude``:
      --metadata data/metadata.tsv \
      --min-date 2012 \
      --exclude exclude.txt \
-     --output-sequences filtered_sequences.fasta \
-     --output-metadata filtered_metadata.tsv
-
-Subsampling within ``augur filter``
-===================================
+     --group-by country year month \
+     --subsample-max-sequences 100 \
+     --output-sequences subsampled_sequences.fasta \
+     --output-metadata subsampled_metadata.tsv
 
```

Comment:
_non-blocking suggestion_

An additional paragraph here, explaining a bit more about what the output is going to look like in different scenarios, would (IMO) be helpful. For example, what happens if `--group-by country year month` results in more than 100 groups? How does that interact with `--subsample-max-sequences 100`?

Reply:
ah, okay, i see the "Caveats" section below addresses this — perhaps a single sentence with a forward pointer (e.g., "See the 'Caveats' section below for some examples of where dataset size and composition can result in getting less than the requested number of outputs.")

Reply:
Good point. The scenario you've described actually isn't covered by the current caveats section. I realized I forgot to include an entire section on probabilistic sampling which would address this scenario. I'll add it.

Reply:
Added: https://github.com/nextstrain/docs.nextstrain.org/blob/96f68da125c4c34d05b5f5fbf41c22bc4b399179/src/guides/bioinformatics/filtering-and-subsampling.rst#L222-L243